_posts/2016/2016-01-15-setup-a-coreos-cluster-just-for-fun-on-centos-and-win.md

```diff
@@ -0,0 +1,262 @@
+---
+layout: post
+title: "Setting up a CoreOS Cluster just for fun on CentOS and Windows"
+date: "2015-12-12 17:17:17"
+icon: fa-cogs
+categories: tech-stories
+authors: ["Andreas Grohmann"]
+---
+
+If you want to play around with [CoreOS][UsingCoreOS] then this article may help you to get started.
+In this article we set up a 3 Cluster CoreOS installation using vagrant and virtual box on Windows and CentOS.
+On windows there are some more things to observe.
+
+# Windows
+
+## Prerequisits
+
+### 1. Power Shell 3.0
+First you have to install the Windows Management Framework 3.0 including the Windows PowerShell 3.0.
+The standard shell will have some troubles with pathes.
+- Download the latest version, e.g. from this link https://www.microsoft.com/en-us/download/details.aspx?id=34595
+
+###  2. install cygwin
+To get the rsync feature you need to install cygwin on your windows installation.
+- You can download it from here https://cygwin.com/install.html
+
+> *Make sure you activate "rsync" and "SSH" in the installation*
+
+- add "C:\cygwin64\bin" to your windows system variable "PATH"
+
+### 3. [VirtualBox](https://www.virtualbox.org/)
+Virtual Box is needed, download latest version from here https://www.virtualbox.org/wiki/Downloads .
```

Comment:
Change to: Virtual Box is required. Download the latest version here. [link auf "here"]
Don't link Virtual Box in the headline but link the first word within the sentence.